tools/interop_matrix/client_matrix.py

```diff
@@ -97,11 +97,10 @@ def get_github_repo(lang):
         'v1.6.6',
     ],
    'csharp': [
-        # Fail to build images due to #13278.
         #'v1.0.1',
-        #'v1.1.4',
-        #'v1.2.5',
-        #'v1.3.9',
+        #'v1.1.4',  Fail to build.
```

Comment:
Are there any issues that can be linked to these errors?

Reply:
I filed bugs internally.